pyRMSD/test/testSymmTools.py

```diff
@@ -58,6 +64,35 @@ def test_symm_group_permutations(self):
         for i, perm in enumerate(symm_permutations(groups)):
             self.assertSequenceEqual(expected_permutations[i], perm)
             
+    def test_symm_group_permutations_new(self):
```

Comment:
No docstring. Should also test the return value when given an empty list of groups.

Reply:
Added